projects/packages/waf/actions.php

```diff
@@ -20,6 +20,15 @@
 	array( __NAMESPACE__ . '\Waf_Runner', 'update_rules_if_changed' )
 );
 
+/**
+ * Cron to update the rules periodically.
+ */
+add_action( 'jetpack_waf_rules_update_cron', array( __NAMESPACE__ . '\Waf_Runner', 'update_rules_cron' ) );
+
+if ( ! wp_next_scheduled( 'jetpack_waf_rules_update_cron' ) ) {
+	wp_schedule_event( time(), 'hourly', 'jetpack_waf_rules_update_cron' );
```

Comment:
Hourly is maybe aggressive; 3 hours? 6 hours? I know we don't have a way to push rules yet, but I suspect that'll be the way we want to tackle rules updates that need to get pushed out right away.

Reply:
@jwidavid The way wp cron works is that the next run always needs to be scheduled, the way it roughly works is that it inserts into the database that a cron needs to be run in X time, if a request comes through after that X time then the cron will be ran alongside that request (it's not a proper cron system that runs in the background), so every request we have to check if the next run is scheduled and if not schedule it again. The downside of the system is that if the site gets no requests then the cron won't run until it does, however I don't think it will hurt for a site without active requests to wait a little longer for updated rules.

Reply:
Changed the cron schedule to twicedaily.